rts/lra/test/basic/src/test/java/io/narayana/lra/arquillian/LRAParticipantAfterLRAIT.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2019, Red Hat, Inc., and individual contributors
```

Comment:
Wrong date. BTW there is a [new copyright statement](https://github.com/jbosstm/narayana/blob/master/CONTRIBUTING.md#legal)